CHANGES/649.misc

```diff
@@ -0,0 +1 @@
+Exclude _frozenlist.cpp from bdists/wheels.
```

Comment:
This probably should go under packaging

Reply:
Yes
https://github.com/aio-libs/frozenlist/blob/master/CHANGES/README.rst

> packaging: Notes for downstreams about unobvious side effects and tooling. Changes in the test invocation considerations and runtime assumptions.

Reply:
I opened https://github.com/aio-libs/frozenlist/pull/650 to address my confusion about the available change types/categories. I’ll rename the change fragment here next.